components/mdx/Layout.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import Layout from '../Layout'
+
+const LessonLayout: React.FC<{}> = ({ children }) => {
```

Comment:
Don't need the empty generics.

```suggestion
const LessonLayout: React.FC = ({ children }) => {
```